bazaarvoice/emodb
# Rename confusing "delta audit" to delta history

## Github Issue #

None

## What Are We Doing Here?

The system of record module of emodb currently refers to delta history as "delta audit" in many locations. This is extremely confusing, as Emo audits are actually completely different from delta history. 

This PR renames all instances where delta history was referred to as audit.

## How to Test and Verify

1. Check out this PR
2. Build and run Emo and ensure that it runs exactly as before

## Risk

### Level 

`Low`

### Required Testing

`Regression`

### Risk Summary

This is extremely low risk, as it is simply renaming existing classes and variables, and not changing functionality in any way.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
